app/components/Table/index.js

```diff
@@ -229,155 +226,131 @@ export default class Table extends Component<Props, State> {
+                      />
+                    </p>
+                  </div>
+                ))}
+                <Button
+                  flat
+                  onClick={() =>
+                    this.setState(
+                      (state) => ({
+                        filters: {
+                          ...state.filters,
+                          [dataIndex]: undefined,
+                        },
+                      }),
+                      () => {
+                        this.toggleFilter(dataIndex);
+                        this.onChange();
+                      }
+                    )
+                  }
+                >
+                  Nullstill
+                </Button>
+              </Dropdown>
             </div>
           )}
           {columnChoices && (
             <div className={styles.arrowDownIcon}>
-              <div
-                ref={(c) => (this.components[dataIndex] = c)}
-                className={styles.arrowDownIcon}
-              >
-                <Icon
-                  onClick={() => this.toggleChooseColumn(dataIndex)}
-                  name="arrow-down"
-                  className={styles.icon}
-                />
-              </div>
-              <Overlay
+              <Dropdown
                 show={isShown[dataIndex]}
-                onHide={() => this.toggleChooseColumn(dataIndex)}
+                toggle={() => this.toggleChooseColumn(dataIndex)}
                 placement="bottom"
-                container={this.container}
-                target={() => this.components[dataIndex]}
+                triggerComponent={
+                  <Icon name="arrow-down" className={styles.icon} />
+                }
+                contentClassName={styles.overlay}
                 rootClose
               >
-                <div className={styles.overlay}>
-                  {columnChoices.map(({ title }, index) => (
-                    <div
-                      key={title}
-                      onClick={() =>
-                        this.onChooseColumnInput(index, dataIndexColumnChoices)
-                      }
-                    >
-                      <p key={title}>
-                        <RadioButton
-                          name={dataIndexColumnChoices}
-                          inputValue={
-                            this.state.showColumn[dataIndexColumnChoices]
-                          }
-                          value={index}
-                          label={title}
-                        />
-                      </p>
-                    </div>
-                  ))}
-                </div>
-              </Overlay>
+                {columnChoices.map(({ title }, index) => (
+                  <div
+                    key={title}
+                    onClick={() =>
+                      this.onChooseColumnInput(index, dataIndexColumnChoices)
+                    }
+                  >
+                    <p key={title}>
```

Comment:
I don't get why `RadioButton` (and `CheckBox` above) must be wrapped inside a paragraph, but I'm sure there once was a great reason and I'll let it slide ... I'm guessing it's because of its default CSS of margin and block display, but I still think it's bad practice.

Reply:
Yea, we should do a proper cleanup of the `Table` component sometime, but I suppose I can do some minor stuff related to the changes I made.

Reply:
Yes, but I suppose a new PR would be fitting 😄

Reply:
It would. I'll leave it like it is for now, and add a larger cleanup to the backlog :)